.github/workflows/python-app.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
```

Comment:
I think it would be interesting to run the workflow on any Push instead of just Main and PR to Main.

`on: push`